tcms/testplans/views.py

```diff
@@ -536,89 +543,12 @@ def clone(request, template_name='plan/clone.html'):
         return HttpResponseRedirect(request.META.get('HTTP_REFERER', '/'))
 
     # Clone the plan if the form is submitted
-    if request.method == "POST":
+    if request.method == "POST":  # pylint: disable=no-else-return
```

Comment:
Don't disable the check, remove the `else` part of this statement.

Reply:
If i move the return outside, the below code will become unreachable.

Reply:
The reason for this is because by doing all the changes below you have modified the behaviour of the function. Now you have:
```

if request.method == 'POST':
    .....
    return ...
else:
   ....
   return ....

....
return
```

before that the else block didn't have a return statement. As with the comment above it may be easier to split this lare function into 2 smaller ones, each of which will handle the GET and POST requests separately; make sure these work and then fix whatever issues are reported by pylint.